tools/crm_resource.c

```diff
@@ -472,6 +472,7 @@ static GOptionEntry query_entries[] = {
     { "list-options", 0, G_OPTION_FLAG_NONE, G_OPTION_ARG_CALLBACK, command_cb,
       "List all available options of the given type\n"
       INDENT "Allowed values:\n"
+      INDENT PCMK__VALUE_PRIMITIVE "(primitive resource meta-attributes), "
       INDENT PCMK_VALUE_FENCING " (parameters common to all fencing resources)",
```

Comment:
any reason just primitive is internal?

Reply:
Yep! `PCMK_VALUE_FENCING` already existed as a public constant for the "requires" meta-attribute. We don't need `PCMK__VALUE_PRIMITIVE` for anything public yet.